python/src/ops.cpp

```diff
@@ -2951,4 +2951,33 @@ void init_ops(py::module_& m) {
         Returns:
           result (array): An array of the same type as ``a`` rounded to the given number of decimals.
       )pbdoc");
+  m.def(
+      "quantized_matmul",
+      &quantized_matmul,
+      "x"_a,
+      "w"_a,
+      "scales"_a,
+      "biases"_a,
+      py::pos_only(),
+      "groups"_a = 128,
+      "width"_a = 4,
+      py::kw_only(),
+      "stream"_a = none,
+      R"pbdoc(
+        quantized_matmul(x: array, w: array, scales: array, biases: array, /, groups: int = 128, width: int = 4, *, stream: Union[None, Stream, Device] = None) -> array
+
+        Perform the matrix multiplication of with the quantized matrix w. The
+        quantization uses one floating point scale and bias per ``groups`` of
+        elements. Each element in w takes ``width`` bits and is packed in q
```

Comment:
What is `q` referring to here?

Reply:
Hehe nothing it is referring to the fact that `q` is close to `a` in the keyboard :-D . Either way it should be "in an unsigned 32-bit integer" I guess.